org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/queues/IQueue.java

```diff
@@ -19,20 +19,20 @@
 import org.eclipse.scanning.api.event.queues.beans.Queueable;
 
 /**
- * IQueue object contain all the information necessary to interact with and 
+ * IQueue object contain all the information necessary to interact with and
  * control a queue within the {@link IQueueService}. IQueue objects contain an
  * {@link IConsumer} to run the queue and all of it's configuration, plus the
  * current {@link QueueStatus}.
- * 
+ *
  * @author Michael Wharmby
  *
- * @param <T> bean object extending {@link Queueable} which is passed around 
- *            this queue. 
+ * @param <T> bean object extending {@link Queueable} which is passed around
+ *            this queue.
  */
 public interface IQueue<T extends Queueable> {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)